data/_ui/interfaces.txt

```diff
@@ -1293,8 +1293,13 @@ interface "bank"
 
 
 interface "mission" bottom
+	visible if "has description"
 	sprite "ui/mission"
 		align bottom
+	visible if "!has description"
+	sprite "ui/mission empty"
+		align bottom
+	visible
 	
```

Comment:
```suggestion

	visible
```
looks better

Reply:
Am I expected to click "Commit suggestion" or is this something that will be done if/when the PR is merged?

Reply:
In this repo usually PR authors commit review suggestions themselves (if they agree to the proposed changes). And, as Amazinite said in #10123, PRs are squash merged, so you don't need to worry about your commit messages.

Reply:
I had missed that reply, thanks. Done now, on both PRs.